router/builders.go

```diff
@@ -187,6 +190,11 @@ func buildKubernetesOutput(sourceSettings *OutputSettings) (*outputs.KubernetesC
 	}, nil
 }
 
+func IsK8s() bool {
```

Comment:
functionality looks like out of scope for `builders.go` 
I would move builder methods to corresponding outputs. (As they are getting bigger with last additions)

Reply:
great point, we should do that for all outputs.

Reply:
https://github.com/aquasecurity/postee/pull/332/commits/ddde6247c8d501fa90a47e0c92093d5165909aab